rocAL_pybind/amd/rocal/types.py

```diff
@@ -62,6 +62,8 @@
 from rocal_pybind.types import NCHW
 from rocal_pybind.types import NFHWC
 from rocal_pybind.types import NFCHW
+from rocal_pybind.types import NONE
```

Comment:
It is better not to expose NONE as layout in the python types

Because it does not signify anything and use might not use it